src/shared/components/ncTable/partials/rowTypePartials/DatetimeDateForm.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<RowFormWrapper :title="column.title" :mandatory="column.mandatory" :description="column.description" :width="2">
-		<NcDatetimePicker v-model="localValue"
+		<NcDateTimePicker v-model="localValue"
```

Comment:
Wasn't mentioned in the migration issues i think but we should move this to https://nextcloud-vue-components.netlify.app/#/Components/NcPickers?id=ncdatetimepickernative for accessibility